packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/FixedEnumSerializationProvider.cs

```diff
@@ -56,32 +61,32 @@ protected override MethodProvider[] BuildMethods()
             // serialization method (in some cases we do not need serialization)
             if (NeedsSerializationMethod())
             {
-                var serializationValueParameter = new ParameterProvider("value", $"The value to serialize.", _enumType.Type);
+                var serializationValueParameter = new ParameterProvider("value", $"The value to serialize.", EnumProvider.Type);
                 var serializationSignature = new MethodSignature(
-                    Name: $"ToSerial{_enumType.ValueType.Name}",
+                    Name: $"ToSerial{EnumProvider.EnumUnderlyingType.Name}",
```

Comment:
Similar comment here: maybe we could build the method name and store it somewhere so that we could use it instead of building it up every time we need it

Reply:
its only used in one place now